package.json

```diff
@@ -31,10 +31,11 @@
     "@nestjs/swagger": "^7.1.1",
     "@prisma/client": "^4.16.1",
     "bcrypt": "^5.1.0",
+    "class-transformer": "^0.5.1",
     "class-validator": "^0.14.0",
     "handlebars": "^4.7.7",
     "nodemailer": "^6.9.3",
-    "paastech-proto": "github:paastech-cloud/proto",
+    "paastech-proto": "github:paastech-cloud/proto#pomegranate",
```

Comment:
Wait for the PR https://github.com/paastech-cloud/proto/pull/8 to be approved and use main branch

Reply:
:+1: